### ISSUE TITLE
Generator sound plays even after crystals are all drained

### ISSUE THREAD

**Issue by mezz** [McJty/DeepResonance#118]
I associate the generator sound with the beam that goes from the energy collector to the crystal nearby.
I think the sound should end when the crystal is empty and the beam is inactive.

**Comment by McJty** [comment]
Yes I know. It is just a tad more complicated then just a simple check though. It has been on my todo for a while :-)

The problem is that the generator is in fact still running because if a pedestal would place a new crystal it would get connected and start generating power anyway. And currently the sound plays when the generator is running regardless of the number of connected crystals
